core/src/main/java/org/opensearch/sql/executor/QueryService.java

```diff
@@ -298,7 +297,7 @@ private FrameworkConfig buildFrameworkConfig() {
             .parserConfig(SqlParser.Config.DEFAULT) // TODO check
             .defaultSchema(opensearchSchema)
             .traitDefs((List<RelTraitDef>) null)
-            .programs(Programs.calc(DefaultRelMetadataProvider.INSTANCE))
+            .programs(Programs.standard())
```

Comment:
use Programs.standard() in building context because the actual execution uses it.